src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -267,6 +269,25 @@ else if (paymentGroup.getDisbursementType().getCode().equals(PdpConstants.Disbur
         }
     }
     
+    private String getCheckYear(PaymentGroup paymentGroup) {
+        String checkYear = KFSConstants.getDashSubAccountNumber();
+        try {
+            CheckReconciliation checkReconciliation = checkReconciliationDao.findByCheckNumber(paymentGroup.getDisbursementNbr().toString());
+            Calendar calendar = Calendar.getInstance();
+            if (ObjectUtils.isNotNull(checkReconciliation) && ObjectUtils.isNotNull(checkReconciliation.getCheckDate())) {
+                calendar.setTime(checkReconciliation.getCheckDate());
+                checkYear = Integer.toString(calendar.get(Calendar.YEAR));
+            }
+            else {
+                LOG.error("getCheckYear", "Failed getting year check was issued, defaulting clearing subaccount to dashes.");
+            }
+        }
+        catch(Exception ex) {
+            LOG.error("getCheckYear", ex);
+        }
+        return StringUtils.isBlank(checkYear) ? KFSConstants.getDashSubAccountNumber() : checkYear;
```

Comment:
since check year is set to KFSConstants.getDashSubAccountNumber() at the top, and then reset lower, is this conditional needed?